packages/common/src/authentication/AuthContext.tsx

```diff
@@ -102,6 +110,7 @@ export const AuthProvider: FC = ({ children }) => {
   const { mutateAsync, isLoading: isLoggingIn } = useGetAuthToken();
   const authCookie = getAuthCookie();
   const [cookie, setCookie] = useState<AuthCookie | undefined>(authCookie);
+  const [error, setError] = useLocalStorage('/auth/error');
```

Comment:
Just trying to clarify my understanding -- is the value of the AuthError in localStorage only used to tell the AuthenticationAlert which message to display?

Reply:
yes - using local storage because I need to retain state and can't access the AuthContext from the Gql provider